src/ipc/dma-copy.c

```diff
@@ -146,7 +146,11 @@ int dma_copy_new(struct dma_copy *dc)
 
 #if !CONFIG_DMA_GW
 	/* get DMA channel from DMAC0 */
+#if CONFIG_RENOIR
+	dc->chan = dma_channel_get(dc->dmac, DMA_TRACE_CHANNEL);
+#else
 	dc->chan = dma_channel_get(dc->dmac, 0);
+#endif
```

Comment:
that should be better abstracted, this doesn't look good to me.

Reply:
maybe we should modify the generic case to remove the hard-coded ß

Reply:
@bhiregoudar yep, best thing to do here is to add a CONFIG_TRACE_CHANNEL (done), add more more info in the Kconfig help section and make all platforms use it (all other platforms will be default to 0).

Reply:
Addressed the Review Comment. Added generic config to reuse by other platforms as well. 
Added more explanation in Kconfig as well.

Reply:
Thanks, but we should change it so everyone uses your new Kconfig option (which defaults to 0 unless set).  So everyone should just use.

```c
dc->chan = dma_channel_get(dc->dmac, CONFIG_DMA_TRACE_CHANNEL);
```